packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -82,10 +84,13 @@ export const Root = styled(FormControl, {
         }
 
         ${ownerState.variant === "horizontal" &&
+        !ownerState.forceVertical &&
         css`
+            container-type: inline-size;
```

Comment:
I believe this is only related to container queries, so it can be removed, right?